packages/core/test/Cardano/types/Address.test.ts

```diff
@@ -20,4 +20,38 @@ describe('Cardano/types/Address', () => {
       'addr1qx52knza2h5x090n4a5r7yraz3pwcamk9ppvuh7e26nfks7pnmhxqavtqy02zezklh27jt9r6z62sav3mugappdc7xnskxy2pn'
     );
   });
+
+  it('Address() accepts a valid testnet grouped address', () => {
```

Comment:
It's using `parseCslAddress`, which has [it's own tests](https://github.com/input-output-hk/cardano-js-sdk/blob/master/packages/core/test/CSL/parseCslAddress.test.ts) for different address types. It would probably be sufficient to test that it calls `isAddress` (which then calls `parseCslAddress`), but I'm fine with having overlapping tests, will be good in case implementation changes in the future.